tests/arithmetic/ast_arithmetic.re

```diff
@@ -0,0 +1,9 @@
+/* Auto-generated by codegen_type */
+type assignment_statement = (variable, string, expression, string)
+and comment = string
+and expression = Intermediate_type1(variable) | Intermediate_type2(number) | Intermediate_type3((expression, string, expression)) | Intermediate_type4((expression, string, expression)) | Intermediate_type5((expression, string, expression)) | Intermediate_type6((expression, string, expression)) | Intermediate_type7((expression, string, expression))
```

Comment:
We can put each of those constructors on their own line (see what I do in ast_grammar.re for example)

Reply:
ok.